sources/core/tests/chat_frame_info_tests.cpp

```diff
@@ -189,21 +193,21 @@ TEST(ChatFrameInfo, PostHear)
 
         info.ApplyHear(&hearer);
         EXPECT_EQ(info.GetPersonalTexts(10).size(), 3);
-        EXPECT_TRUE(info.GetPersonalTexts(10).contains("human1:text1"));
-        EXPECT_TRUE(info.GetPersonalTexts(10).contains("human2:text2"));
-        EXPECT_TRUE(info.GetPersonalTexts(10).contains("human3:text3"));
+        EXPECT_TRUE(info.GetPersonalTexts(10).contains("human1:text1:express1"));
```

Comment:
You have changed checks, but you didn't change how heard messages are formatted here, and tests are expectedly failed. It is a good idea to run tests locally after each change (especially in tests) in order to check if they are still passing.
Here you need to fix `TestHearer::Hear` function https://github.com/crazzymad777/griefly/blob/54eb2f7be6608e079986bb8fc95599dd4f3a8a3e/sources/core/tests/chat_frame_info_tests.cpp#L149 
It should use `phrase.expression`.